src/app/routes.php

```diff
@@ -5,10 +5,15 @@
 * @var array
 */
 $routes = [
-    "/trash/"   => "/jugaad/trash/",
-    "/jugaad/"  => "/jugaad/read/",
-    "/ajax/"    => "/ajax/",
-    "/auth/"    => "/auth/",
+    "/trash/"       => "/jugaad/trash/",
+    "/jugaad/"      => "/jugaad/read/",
+    "/ajax/"        => "/ajax/",
+    "/auth/"        => "/auth/",
+
+    "/login/"       => "/static_page/login/",
```

Comment:
We could also have `"/login/"       => "/auth/login/",`, we prefer this PR over that?